linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/ansible/shared.yml

```diff
@@ -5,15 +5,15 @@
 # disruption = medium
 - name: "Enable GNOME3 Login Warning Banner"
   ini_file:
-    dest: "/etc/dconf/db/local.d/00-security-settings"
+    dest: "/etc/dconf/db/gdm.d/00-security-settings"
     section: "org/gnome/login-screen"
     option: banner-message-enable
     value: "true"
     create: yes
 
 - name: "Prevent user modification of GNOME banner-message-enabled"
   lineinfile:
-    path: /etc/dconf/db/local.d/locks/00-security-settings-lock
+    path: /etc/dconf/db/gdm.d/locks/00-security-settings-lock
```

Comment:
This lock mechanism doesn't work when the dconf is set to use text backend

Reply:
As of 23rd of May, we are not sure about this. We are in contact with dconf experts, who also have to look things up before they tell us how we are.